EventsExpress/ClientApp/src/actions/register/register-complete-action.js

```diff
@@ -1,32 +1,47 @@
+import { SubmissionError } from 'redux-form';
+import moment from 'moment';
+import jwt from 'jsonwebtoken';
 import { AuthenticationService } from '../../services';
-import { createBrowserHistory } from 'history';
-import { setSuccessAllert } from '../alert-action';
+import { setSuccessAllert, setErrorAllertFromResponse } from '../alert-action';
 import { buildValidationState } from '../../components/helpers/action-helpers';
-import { SubmissionError } from 'redux-form';
 import { jwtStorageKey } from '../../constants/constants';
-import jwt from 'jsonwebtoken';
+import { getUserInfo } from '../login/login-action';
 
 const api_serv = new AuthenticationService();
-const history = createBrowserHistory({ forceRefresh: true });
 
-export default function registerComplete(data) {
+export default function registerComplete(data, { shouldSaveMoreInfo }) {
     return async dispatch => {
-        data.accountId = getAccountIdFromJWT()
+        const body = {
+            ...data,
+            birthday: moment.utc(data.birthday).local().format('YYYY-MM-DD[T00:00:00]'),
+            accountId: getAccountIdFromJWT(),
```

Comment:
Why do we need to pass in accountId? Couldn't we get it on the server from token?

Reply:
I think we can do it on the server side. I will adjust this